testData/solutions/win32.json5

```diff
@@ -4977,7 +4977,8 @@
                                     "default": "%SystemRoot%\\Web\\Wallpaper\\Windows\\img0.jpg"
                                 }
                             }
-                        }
+                        },
+                        "path": "pvParam"
```

Comment:
We should maybe discuss this in our next meeting, I think with a little work this could be a very positive change.  Right now, the schema still allows path, but you'd in essence ignore that and clobber it with whatever was defined as an "option" outside of the schema.  That seems a bit dishonest, we should not advertise `path` as anything the user can control if we're gonna ignore them.

I'd argue that if you're gonna do this, you should go ahead collapse the `value` property as well, so that the transforms can either be removed or just become a simple map of (legacy) user-facing settings to settings-handler-facing settings.